### PR TITLE
afc 1306 [ADD] Move jobseeker module, also rename it

### PR DESCRIPTION
**Pull request by arturchilangwa-vertel**

### CODE REVIEW

File: outplacement/models/outplacement.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
+import base64
+import logging
+
+from odoo import api, fields, models, tools, SUPERUSER_ID
+from odoo import tools, _
+from odoo.exceptions import ValidationError, AccessError
+from odoo.modules.module import get_module_resource
+
+
+class JobSeeker(models.Model):
```
Comment (nils-vertel) [inline outplacement/models/outplacement.py]: Rename this?

File: outplacement/views/outplacement_view.xml [hunk 2]
```diff
@@ -0,0 +1,138 @@
+<odoo>
+    <data>
+        <record id="outplacement_form_view" model="ir.ui.view">
+            <field name="name">HR Job Seeker</field>
```
Comment (nils-vertel) [inline outplacement/views/outplacement_view.xml]: Rename?

File: outplacement/views/templates.xml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <template id="hr_assets_backend" name="Job Seeker Assets" inherit_id="web.assets_backend">
+            <xpath expr="." position="inside">
+                <link rel="stylesheet" type="text/scss" href="/hr_jobseeker/static/src/scss/job_seeker.scss"/>
```
Comment (nils-vertel) [inline outplacement/views/templates.xml]: rename?

**Review by nils-vertel — CHANGES_REQUESTED** [review]
Check on comments

File: outplacement/views/outplacement_view.xml (hunk 2 above)
Comment (arturchilangwa-vertel) [reply outplacement/views/outplacement_view.xml]: Fixed in 27874a8

File: outplacement/models/outplacement.py (hunk 1 above)
Comment (arturchilangwa-vertel) [reply outplacement/models/outplacement.py]: Fixed in 27874a8

File: outplacement/views/templates.xml (hunk 3 above)
Comment (arturchilangwa-vertel) [reply outplacement/views/templates.xml]: Fixed in 27874a8

**Review by nils-vertel — APPROVED** [review]

**Pull request merged**